.github/workflows/focus-ios-ui-tests.yml

```diff
@@ -117,10 +117,10 @@ jobs:
               id: slack
               uses: slackapi/slack-github-action@v1.26.0
               with:
+                channel-id: ${{ secrets.CHANNEL_ID }}
                 payload-file-path: ${{ env.browser }}/slack.json
               env:
-                SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
-                SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+                SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
```

Comment:
I'm guessing that what I need to do. 🤞🏼 
https://github.com/slackapi/slack-github-action